frontend/src/metabase/lib/formatting/date.unit.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import { formatDateTimeRangeWithUnit } from "metabase/lib/formatting/date";
+import { OptionsType } from "metabase/lib/formatting/types";
+
+describe("date range formatting", () => {
```

Comment:
Nit: usually we just name the block the same as the function it is testing

```typescript
describe("formatDateTimeRangeWithUnit", () => {
```

Reply:
44ec5b2a9d251d2cc074295908258799937b7fdc